src/components/__tests__/StackBar.test.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import StackBar from '../StackBar';
+import renderer from 'react-test-renderer';
+
+describe('StackBar chart', () => {
+  test('renders StackBar component with 2d array data format', () => {
+    const data = [
+      ['product', '2015', '2016', '2017'],
+      ['Matcha Latte', 43.3, 85.8, 93.7],
+      ['Milk Tea', 83.1, 73.4, 55.1],
+      ['Cheese Cocoa', 86.4, 65.2, 82.5],
+      ['Walnut Brownie', 72.4, 53.9, 39.1]
+    ];
+
+    const config = {
+      title: 'StackBar Chart Example',
+      xAxisLabel: 'Categories',
+      yAxisLabel: 'Values'
+    };
+
+    render(
+      <StackBar
+        config={config}
+        data={data}
+      />
+    );
+
+    const chartContainer = screen.getByRole('figure');
+    expect(chartContainer).toBeInTheDocument();
```

Comment:
The test passed, but when I tried this with @ifirmawan, the chart was blank. Iwan has already implemented this in the example, which is why we merged the recent example features here. Please make the necessary changes to make this work in the example.